### ISSUE TITLE
Relax engine requirements

### ISSUE THREAD

**Issue by lee-reinhardt** [praneshr/react-diff-viewer#59]
The latest version 3.0.0 looks great, and thank you for all the improvements.

However, one thing that changed is the `engines` definition in `package.json`. When attempting to install with yarn, it refuses to install if you're not using node 12.

This can be worked around with the `--ignore-engines` flag, but it's not clear whether requiring v12 is absolutely necessary. I think at least v10 should be allowed as well.

```
$ yarn add react-diff-viewer
yarn add v1.21.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error react-diff-viewer@3.0.0: The engine "node" is incompatible with this module. Expected version "^12". Got "10.17.0"
error Found incompatible module.
```

**Comment by praneshr** [comment]
@lee-reinhardt Thanks for pointing this out. I’ll release a patch ASAP. Till then please use `—ignore-engines`

**Comment by praneshr** [comment]
Fixed in [v3.0.1](https://github.com/praneshr/react-diff-viewer/releases/tag/v3.0.1)

**Comment by simmerer** [comment]
Just experienced this issue with 3.0.1, sadly:

```
$ yarn add react-diff-viewer
yarn add v1.21.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error react-diff-viewer@3.0.1: The engine "node" is incompatible with this module. Expected version "^12 || ^10 || ^8". Got "13.6.0"
error Found incompatible module.
```
